data-raw/process_data_sets_meredith.R

```diff
@@ -0,0 +1,143 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+
+# ahca_polls ----------------------------------------------------------------------
+ahca_polls <- read_csv("data-raw/ahca-polls/ahca_polls.csv")
+# snake_case for variable names
+colnames(ahca_polls) <- colnames(ahca_polls) %>%
+  tolower()
+# Change start and end dates to proper format
+ahca_polls$start <- as.Date(ahca_polls$start, "%m/%d/%y") 
+ahca_polls$end <- as.Date(ahca_polls$end, "%m/%d/%y")
+# Convert to factor variable
+ahca_polls$pollster <- as.factor(ahca_polls$pollster)
```

Comment:
See my comment on line 93 and 210 of Maggie's pull request [here](https://github.com/rudeboybert/fivethirtyeight/pull/14/files/a18deaf335870d6fa65d3ea23ab1f5ab56903d60?diff=split) about having everything in a single chain of `%>%` instead of individual lines of operations. It really helps make the code legible and less cluttered.

For example this data set would be:
```
ahca_polls <- read_csv("data-raw/ahca-polls/ahca_polls.csv") %>% 
  clean_names() %>% 
  mutate(
    start = as.Date(start, "%m/%d/%y"),
    end = as.Date(end, "%m/%d/%y"),
    pollster = as.factor(pollster)
  )
devtools::use_data(ahca_polls, overwrite = TRUE)
```